From 2c3869495da2c4dba545d219013e9434e49ce472 Mon Sep 17 00:00:00 2001
From: Harninder Rai <harninder.rai@freescale.com>
Date: Fri, 31 May 2013 15:20:34 +0530
Subject: [PATCH 371/547] eSDHC: Revert: Remove Quirks for Multi block read and
 timeout

For BSC9132QDS platform, once Quirk related strings are corrected
in device tree, the current mmc driver takes care of setting
them.

Signed-off-by: Harninder Rai <harninder.rai@freescale.com>
Signed-off-by: Vijay Rai <vijay.rai@freescale.com>
Signed-off-by: Naveen Burmi <naveenburmi@freescale.com>
Change-Id: I46eb23eb234b40e0656637ce85d62debc0633a43
Reviewed-on: http://git.am.freescale.net:8181/2811
Reviewed-by: Zhang Haijun-B42677 <haijun.zhang@freescale.com>
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Xie Xiaobo-R63061 <X.Xie@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
[Original patch taken from QorIQ-SDK-V1.4-SOURCE-20130625-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/mmc/host/sdhci-of-esdhc.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/mmc/host/sdhci-of-esdhc.c b/drivers/mmc/host/sdhci-of-esdhc.c
index 9493920..b0dc026 100644
--- a/drivers/mmc/host/sdhci-of-esdhc.c
+++ b/drivers/mmc/host/sdhci-of-esdhc.c
@@ -417,9 +417,7 @@ static const struct sdhci_pltfm_data sdhci_esdhc_pdata = {
 	 */
 	.quirks = ESDHC_DEFAULT_QUIRKS | SDHCI_QUIRK_BROKEN_CARD_DETECTION
 		| SDHCI_QUIRK_NO_CARD_NO_RESET
-		| SDHCI_QUIRK_NO_ENDATTR_IN_NOPDESC
-		| SDHCI_QUIRK_BROKEN_TIMEOUT_VAL
-		| SDHCI_QUIRK_MULTIBLOCK_READ_ACMD12,
+		| SDHCI_QUIRK_NO_ENDATTR_IN_NOPDESC,
 	.ops = &sdhci_esdhc_ops,
 };
 
-- 
1.8.4.93.g57e4c17

